nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -169,6 +143,13 @@
         .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
         .build();
 
+    protected static final PropertyDescriptor IGNORE_NULL = new Builder()
+        .name("Ignore NULL")
+        .description("Ignore NULL on Kudu Put Operation, Update only non-Null columns if set true")
+        .defaultValue("true")
```

Comment:
since this is a new property was it effectively 'false' by default previously?  If so, to maintain backward compatibility we should set this to false.

Reply:
@joewitt IGNORE_NULL only introduced in this PR, you can check master branch file here 
https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

Reply:
done!